src/index.tsx

```diff
@@ -1,7 +1,5 @@
-import {
-  createTheme, Paper,
-  ThemeProvider
-} from '@mui/material'
+import Paper from '@mui/material/Paper'
+import { createTheme, ThemeProvider } from '@mui/material/styles'
```

Comment:
@anthonyalayo is this expected?
maybe I mess it up in the rebasing 🤔 

use this instead?
```js
import createTheme from '@mui/material/styles/createTheme'
import ThemeProvider from '@mui/material/styles/ThemeProvider'
```

Reply:
@pionxzh looks like it's still using cjs for some reason?

```
10:41:43 PM: /opt/build/repo/node_modules/@mui/material/styles/createTheme.js:1
10:41:43 PM: import _extends from "@babel/runtime/helpers/esm/extends";
10:41:43 PM: ^^^^^^
10:41:43 PM: SyntaxError: Cannot use import statement outside a module
10:41:43 PM:     at internalCompileFunction (node:internal/vm:74:18)
10:41:43 PM:     at wrapSafe (node:internal/modules/cjs/loader:1141:20)
10:41:43 PM:     at Module._compile (node:internal/modules/cjs/loader:1182:27)
10:41:43 PM:     at Module._extensions..js (node:internal/modules/cjs/loader:1272:10)
10:41:43 PM:     at Module.load (node:internal/modules/cjs/loader:1081:32)
10:41:43 PM:     at Module._load (node:internal/modules/cjs/loader:922:12)
10:41:43 PM:     at Module.require (node:internal/modules/cjs/loader:1105:19)
10:41:43 PM:     at require (node:internal/modules/cjs/helpers:103:18)
10:41:43 PM:     at 3898 (/opt/build/repo/docs/.next/server/pages/index.js:726:18)
```

Reply:
it's quite weird... 🤔  still finding the root cause